PIL/TiffImagePlugin.py

```diff
@@ -230,7 +241,12 @@
     (MM, 8, (1,), 1, (8, 8, 8), ()): ("LAB", "LAB"),
 }
 
-PREFIXES = [b"MM\000\052", b"II\052\000", b"II\xBC\000"]
+PREFIXES = [
+    b"MM\x00\x2A",  # Valid TIFF header with big-endian byte order
+    b"II\x2A\x00",  # Valid TIFF header with little-endian byte order
+    b"MM\x2A\x00",  # Invalid TIFF header, assume big-endian
+    b"II\x00\x2A",  # Invalid TIFF header, assume little-endian
```

Comment:
According to [the specs](https://www.itu.int/itudoc/itu-t/com16/tiff-fx/docs/tiff6.pdf) (page 13), there are only two valid headers: `"MM\x00*"` and `"II*\x00"`. But I met files with wrong headers, where second two bytes were not aligned according to the reported ending. However, these files were successfully opened by other viewers and were absolutely correct in anything else.

As of `"II\xBC\x00"` prefix, I have no Idea what does it mean. I haven't find it in specs or anywhere else in web. It was there from original PIL and I don't know original motivation of adding it.